arm-devtestlabs/2016-05-15/swagger/DTL.json

```diff
@@ -56,26 +63,78 @@
                 "x-ms-odata": "#/definitions/Lab"
             }
         },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.DevTestLab/schedules": {
+            "get": {
+                "tags": [ "GlobalSchedule" ],
+                "description": "List schedules in a subscription.",
+                "operationId": "GlobalSchedule_ListBySubscription",
+                "parameters": [
+                    { "$ref": "#/parameters/subscriptionId" },
+                    {
+                        "name": "$expand",
+                        "in": "query",
+                        "description": "Specify the $expand query. Example: 'properties($select=status)'",
+                        "required": false,
+                        "type": "string"
+                    },
+                    {
+                        "name": "$filter",
+                        "in": "query",
+                        "description": "The filter to apply to the operation.",
+                        "required": false,
+                        "type": "string"
+                    },
+                    {
+                        "name": "$top",
+                        "in": "query",
+                        "description": "The maximum number of resources to return from the operation.",
+                        "required": false,
+                        "type": "integer",
+                        "format": "int32"
+                    },
+                    {
+                        "name": "$orderBy",

```

Comment:
Not sure which version of OData semantics team is following but I found something http://docs.oasis-open.org/odata/odata/v4.0/errata03/os/complete/part2-url-conventions/odata-v4.0-errata03-os-part2-url-conventions-complete.html#_Toc453752361 for reference.